### DISCUSSION TITLE
v11 to v16 in two months?

### DISCUSSION THREAD

**Discussion by sfkpmr** [motdotla/dotenv#630] [Q&A] [answered]
How come it reached v16 so rapidly instead of say 11.X.X?

**Comment by ckcr4lyf** [comment]
Due to breaking changes, and I assume its following semver (Semantic Versioning)

https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md

**Reply by sfkpmr** [reply]
I suppose. Would love to hear what those changes would have been.

**Reply by ckcr4lyf** [reply]
The changes are _in_ the changelog

**Comment by motdotla** [comment, marked as answer]
Indeed it was because of breaking changes. I agree it was a lot in a short amount of time. I apologize for that. We're rolling out more slowly going forward.
